xml/System/Type.xml

```diff
@@ -2663,7 +2663,7 @@ False
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Type.GetConstructors%2A> method does not return constructors in a particular order, such as declaration order. Your code must not depend on the order in which constructors are returned, because that order varies.  
+ In .NET 6 and below versions the <xref:System.Type.GetConstructors%2A> method does not return constructors in a particular order, such as declaration order. Your code must not depend on the order in which constructors are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```

Comment:
```suggestion
 In .NET 6 and earlier versions, the <xref:System.Type.GetConstructors%2A> method does not return constructors in a particular order, such as declaration order. Your code must not depend on the order in which constructors are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```